e2e/testdata/attachments.dockerapp/nesteddir/config2.cfg

```diff
@@ -0,0 +1 @@
+Some config
```

Comment:
Should have different data in each config file to ensure the correct file is in the correct place?